guardian/manage-frontend
# Confirm Cancellation V1

## What does this change?
New screen which the user can confirm whether they would like to confirm to cancel their membership.
Button on offers page changed to link to this new page as per flow on Figma.



## How to test
Storybook added or switch feature flag on and head to cancel/membership/confirm on local


## How can we measure success?



## Have we considered potential risks?



## Images
![image](https://user-images.githubusercontent.com/115992455/235135846-42fd56ea-b0bd-4ee5-8aba-fc9f3aa7f292.png)



## Accessibility
